main.cxx

```diff
@@ -47,8 +47,8 @@ static inline bool handle_raw_input_2(RAWINPUT const& raw)
 		       (long int)raw.data.mouse.lLastY,
 		       (unsigned long int)raw.data.mouse.ulExtraInformation);
 		if (raw.data.mouse.usButtonFlags == RI_MOUSE_WHEEL)
-		    printf("usButtonFlags: RI_MOUSE_WHEEL usButtonData: %ui\n",
-		           (unsigned int)raw.data.mouse.usButtonData);
+		    printf("usButtonFlags: RI_MOUSE_WHEEL usButtonData: %dsi\n",
```

Comment:
What's the "si" after "%d" for?

Reply:
I was trying to copy what you were doing without really understanding it. It's not for anything in particular.